src/ApiCheck/ApiListing/ApiListingFilters.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using ApiCheck.Description;
+
+namespace ApiCheck
+{
+    public static class ApiListingFilters
+    {
+        public static bool NonExportedMembers(MemberInfo e)
```

Comment:
Predicates usually have some interrogative question in them, e.g. `HasNonExportedMember` - is that what is meant here? (And the other filters?)